src/component/Board.jsx

```diff
@@ -25,11 +24,15 @@ const createCards = (cardValue) => {
   }
   // console.log(card);
 };
-
 createCards(cardValue);
 
 function Board() {
   // createCards(cardValue);
+  const [, forceUpdate] = useReducer((x) => x + 1, 0);
+
+  function update() {
+    forceUpdate();
```

Comment:
Why we need forceupdate?

Reply:
after comparing to the card it's not re-rendering the card component as per new data. so forceUpdate is helping to re-render the card component.

Reply:
Removed forceupdate